docs/extended_api/shapes.md

```diff
@@ -1,17 +1,10 @@
----
-parent: Extended API
-has_children: true
-has_toc: false
-nav_order: 2
----
+## Shapes
 
-# Shapes
-
-| [`cuda::std::size_t`]    | Defines an extent of bytes. `(typedef)`                                            |
-| [`cuda::aligned_size_t`] | Defines an extent of bytes with a statically defined alignment. `(class template)` |
+| [`cuda::std::size_t`]    | Defines an extent of bytes. `(typedef)`                                            <br/><br/> 1.0.0 / CUDA 10.2 |
+| [`cuda::aligned_size_t`] | Defines an extent of bytes with a statically defined alignment. `(class template)` <br/><br/> 1.2.0 / CUDA 11.1 |
 
 
 [`cuda::std::size_t`]: https://en.cppreference.com/w/cpp/types/size_t
 
-[`cuda::aligned_size_t`]: ./shapes/aligned_size_t.md
+[`cuda::aligned_size_t`]: {{ "shapes/aligned_size_t.html" | relative_url }}
```

Comment:
This ends up missing the `extended_api` path segment.

Reply:
Fixed.